lib/runner.js

```diff
@@ -265,10 +265,17 @@ Runner.prototype.fail = function(test, err) {
  * @param {Error} err
  */
 Runner.prototype.failHook = function(hook, err) {
+  hook.originalTitle = hook.originalTitle || hook.title;
   if (hook.ctx && hook.ctx.currentTest) {
-    hook.originalTitle = hook.originalTitle || hook.title;
     hook.title =
       hook.originalTitle + ' for "' + hook.ctx.currentTest.title + '"';
+  } else {
+    var parentTitle = hook.parent.title
+      ? hook.parent.title
+      : hook.parent.root
+      ? '{root}'
+      : '';
```

Comment:
I asked before... please reformulate this. Since prettier denies us the ability to indent the nested ternary, we should not use them. Nesting them is considered a code smell in many languages. [This article](https://medium.com/javascript-scene/nested-ternaries-are-great-361bddd0f340) says they're great, but almost _all_ the commenters disagree.

```js
    var parentTitle;
    if (hook.parent.title) {
      parentTitle = hook.parent.title;
    } else {
      parentTitle = hook.parent.root
        ? '{root}'
        : '';
    }
```

Reply:
@plroebuck I usually expect more tolerance against other's coding style, but I will make this change for you. This PR was my first one and you have supported me from beginning. So thanks for that!
Tomorrow I'm going to merge this PR.

Reply:
agree. @mochajs/core 	please dial it back on code style comments.  our linters and formatters are more than up to the task.  if they fall short, we can create new rules to extend functionality.